scripts/cc-cli/src/commands/unbond.ts

```diff
@@ -6,11 +6,16 @@ import { getStatus, requireStatus } from "../utils/status";
 
 export function makeUnbondCommand() {
   const cmd = new Command("unbond");
-  cmd.description("Schedule a portion of the stash to be unlocked");
-  cmd.option("-s, --seed [mnemonic]", "Specify mnemonic phrase to use");
+  cmd.description(
+    "Schedule a portion of the stash to be unlocked by the controller"
+  );
+  cmd.option(
+    "-s, --seed [mnemonic]",
+    "Specify controller mnemonic phrase to use"
```

Comment:
```suggestion
    "-cs, --controller-seed [controller-seed]",
    "Specify mnemonic phrase to use for controller account"
```

Same as `wizard` command.